catalog/schema/workflows.yaml

```diff
@@ -45,3 +49,11 @@ classes:
         description: The description of the workflow.
         required: true
         range: string
+      parameters:
+        description: The parameters of the workflow. Dictionary with arbitrary depth, string keys and primitive values.
+        required: false
+        range: Any
```

Comment:
idk linkml almost at all, but im finding it surprising there isnt a way to do what we want here